crates/red_knot_python_semantic/resources/mdtest/comparison/byte_literals.md

```diff
@@ -0,0 +1,43 @@
+### Comparison: Byte literals
+
+These tests assert that we infer precise `Literal` types for comparisons between objects
+inferred as having `Literal` bytes types:
+
+```py
+reveal_type(b"abc" == b"abc")  # revealed: Literal[True]
+reveal_type(b"abc" == b"ab")   # revealed: Literal[False]
```

Comment:
> Does it work without `reveal_type`?

no, the `reveal_type` call is necessary to trigger a "Revealed" assertion when using `mdtest`-based tests. This is documented [here](https://github.com/astral-sh/ruff/tree/main/crates/red_knot_test#assertions).

> Is comment aligning part of mdtest's design guide? (is it allowed or preferred?)

The comment alignment seems fine to me. At some point maybe we might start running our `check_docs_formatted.py` CI script on our `mdtest` test files. But until we do that, or introduce autoformatting of these Python snippets some other way. I don't think we should be too picky about the formatting of the Python snippets in our test files.

Reply:
This is probably something we have to align on. I recommend to not rely on comment alignment because it's something that will break if we ever decide to format our example code using ruff (which I think would be a nice addition).